docs/docs/pods.md

```diff
@@ -0,0 +1,572 @@
+        {
+          "name": "v1",
+          "mountPath": "jdef-v1"
+        }
+      ]
+    },
+    {
+      "name": "ct2",
+      "resources": {
+        "cpus": 0.1,
+        "mem": 32
+      },
+      "exec": { "command": { "shell": "while true; do cat ./etc/clock; sleep 1; done" } },
+      "volumeMounts": [
+        {
+          "name": "v1",
+          "mountPath": "etc"
+        }
+      ]
+    }
+  ],
+  "networks": [
+    { "mode": "host" }
+  ],
+  "volumes": [
+    { "name": "v1" }
+  ]
+}
+```
+
+## IP-per-Pod Networking
+
+The following pod definition specifies a virtual (user) network named `my-virtual-network-name`.
+
+```json
+{
+  "id": "/pod-with-virtual-network",
+  "scaling": { "kind": "fixed", "instances": 1 },
+  "containers": [
+    {
+      "name": "sleep1",
+      "exec": { "command": { "shell": "sleep 1000" } },
+      "resources": { "cpus": 0.1, "mem": 32 }
+    }
+  ],
+  "networks": [ { "mode": "container", "name": "my-virtual-network-name" } ]
+}
+```
+
+This pod declares a “web” endpoint that listens on port 80.
+
+```json
+{
+  "id": "/pod-with-endpoint",
+  "scaling": { "kind": "fixed", "instances": 1 },
+  "containers": [
+    {
+      "name": "sleep1",
+      "exec": { "command": { "shell": "sleep 1000" } },
+      "resources": { "cpus": 0.1, "mem": 32 },
+      "endpoints": [ { "name": "web", "containerPort": 80, "protocol": [ "http" ] } ]
+    }
+  ],
+  "networks": [ { "mode": "container", "name": "my-virtual-network-name" } ]
+}
+```
+
+This pod adds a health check that references the “web” endpoint; mesos will execute an HTTP request against `http://<master-ip>:80/ping`.
+
+```json
+{
+  "id": "/pod-with-healthcheck",
+  "scaling": { "kind": "fixed", "instances": 1 },
+  "containers": [
+    {
+      "name": "sleep1",
+      "exec": { "command": { "shell": "sleep 1000" } },
+      "resources": { "cpus": 0.1, "mem": 32 },
+      "endpoints": [ { "name": "web", "containerPort": 80, "protocol": [ "http" ] } ],
+      "healthCheck": { "http": { "endpoint": "web", "path": "/ping" } }
+    }
+  ],
+  "networks": [ { "mode": "container", "name": "my-virtual-network-name" } ]
+}
+```
+
+## Complete Pod 
+The following pod definition can serve as a reference to create more complicated pods. Information about the different properties can be found in the documentation for Marathon applications.
+
+```
+{
+  "id": "/complete-pod",
+  "labels": {
+    "owner": "zeus",
+    "note": "Away from olympus"
+  },
+  "environment": {
+    "XPS1": "Test"
+  },
+  "volumes": [
+    {
+      "name": "etc",
+      "host": "/hdd/tools/docker/registry"
+    }
+  ],
+  "networks": [
+    {
+      "mode": "host"
+    }
+  ],
+  "scaling": {
+    "kind": "fixed",
+    "instances": 1
+  },
+  "scheduling": {
+    "backoff": {
+      "backoff": 1,
+      "backoffFactor": 1.15,
+      "maxLaunchDelay": 3600
+    },
+    "upgrade": {
+      "minimumHealthCapacity": 1,
+      "maximumOverCapacity": 1
+    },
+    "placement": {
```

Comment:
please don't include fields that don't include data. for example, `placement` is completely useless here